src/js/media-controller.js

```diff
@@ -663,8 +694,9 @@ const getMediaUIAttributesFrom = (child) => {
   );
 };
 
-const isMediaStateReceiver = (child) =>
-  !!getMediaUIAttributesFrom(child).length;
+const isMediaStateReceiver = (child) => {
+  return !!getMediaUIAttributesFrom(child).length;
```

Comment:
needed a breakpoint? 😄

Reply:
Ah, didn't mean to leave this changed, but yes! Somewhere to drop a console.log.